### PR TITLE
Add trigger and condition to configuration vars

### PR DESCRIPTION
**Pull request by TheFes**
Whoops, forgot two important ones

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation/yaml.markdown [hunk 1]
```diff
@@ -84,6 +84,28 @@ max_exceeded:
   required: false
   type: string
   default: warning
+trigger:
+  description: The trigger(s) which will start the automation. Multiple triggers can be added and the automation will start when any of these triggers changes from `false` to `true`.
```
Comment (frenck) [inline source/_docs/automation/yaml.markdown]: Triggers are not conditional (e.g., can be event based too); so its not about being true or false.

```suggestion
  description: The trigger(s) which will start the automation. Multiple triggers can be added and the automation will start when any of these triggers trigger.
```

File: source/_docs/automation/yaml.markdown [hunk 2]
```diff
@@ -84,6 +84,28 @@ max_exceeded:
...
+  required: true
+  type: list
+  keys:
+    id:
+      description: An id which can be used in the automatin to determine which trigger caused the automation to start
```
Comment (frenck) [inline source/_docs/automation/yaml.markdown]:
```suggestion
      description: An ID that can be used in the automation to determine which trigger caused the automation to start.
```

File: source/_docs/automation/yaml.markdown [hunk 3]
```diff
@@ -84,6 +84,28 @@ max_exceeded:
...
+      type: string
+      required: false
+    variables:
+      description: Variables which will be available in the conditions and action sequence.
```
Comment (frenck) [inline source/_docs/automation/yaml.markdown]:
```suggestion
      description: Variables that will be available in the conditions and action sequence.
```

File: source/_docs/automation/yaml.markdown [hunk 4]
```diff
@@ -84,6 +84,28 @@ max_exceeded:
...
+      required: false
+      default: {}
+      type: map
+      keys:
+        PARAMETER_NAME:
+          description: The value of the variable. Any YAML is valid. Templates can also be used to pass a value to the variable.
+          type: any
+condition:
+  description: Conditions which have to be `true` to start the automation. By default all conditions listed have to be `true`, you can use [logical conditions](https://www.home-assistant.io/docs/scripts/conditions/#logical-conditions) to change this default behaviour.
```
Comment (frenck) [inline source/_docs/automation/yaml.markdown]:
```suggestion
  description: Conditions that have to be `true` to start the automation. By default all conditions listed have to be `true`, you can use [logical conditions](/docs/scripts/conditions/#logical-conditions) to change this default behavior.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by TheFes** [comment]
@frenck I think I messed something up, It seems there is a review requesting changes pending, but I don't see it anymore.

**Review by klaasnicolaas — APPROVED** [review]
✅ LGTM but let @frenck make the final review 👀 
BTW the open change request is because of the first round of changes that Frenck has suggested 😉

**Comment by TheFes** [comment]
@klaasnicolaas Thanks! I accepted all of them, but then messed something up and committed a version without the new part in it.
I did a new commit after that, with the `trigger` and  `condition` part in it, including the proposed changes by @frenck 

But if it is just the final review now, then it's probably fine. :)

**Review by frenck — APPROVED** [review]
Thanks, @TheFes 👍

**Pull request merged**